docs/CONTRIBUTOR_GUIDE.md

```diff
@@ -0,0 +1,55 @@
+# Contributor Guide
+
+This guide is intended for people who want to start working on `krew` itself.
+If you intend to write a new plugin, see [Developer Guide](./DEVELOPER_GUIDE.md) instead.
+
+## Setting up the environment
+
+Krew is built with go 1.10, but newer versions will do as well.
+Most toolchains will expect that the krew repository is on the `GOPATH`.
+To set it up correctly, do
+```bash
+mkdir -p $(go env GOPATH)/src/sigs.k8s.io/krew
+cd $(go env GOPATH)/src/sigs.k8s.io/krew
+git clone https://github.com/kubernetes-sigs/krew .
+git remote set-url origin --push no_push   # to avoid pushes
+```
+
+## Code style
+
+Krew adheres to standard `golang` code formatting conventions, and also expects imports sorted properly.
+To automatically format code appropriately, run
+```bash
+goimports -w cmd pkg
```

Comment:
I suspect this won't be in $PATH.

Why don't we just tell `gofmt -s -w ./...`

Reply:
`gofmt` does not group the imports as desired. But I should include some instructions how to set up `goimports`.

Reply:
And the reason to not do `goimports -w .` is that this would process the `vendor/` folder which does more harm than good.